react/src/lib/components/DeckGLMap/components/InfoCard.tsx

```diff
@@ -0,0 +1,201 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Collapse from "@material-ui/core/Collapse";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableContainer from "@material-ui/core/TableContainer";
+import TableRow from "@material-ui/core/TableRow";
+import Paper from "@material-ui/core/Paper";
+
+import { Button, Icon } from "@equinor/eds-core-react";
+import { arrow_drop_up, arrow_drop_down } from "@equinor/eds-icons";
+
+import { PickInfo } from "deck.gl";
+import { PropertyMapPickInfo } from "../layers/utils/propertyMapTools";
+
+Icon.add({ arrow_drop_up, arrow_drop_down });
+
+export interface LayerPickInfo extends PickInfo<unknown> {
+    properties?: PropertyDataType[];
```

Comment:
Does this have to be an array? Afaik, the multipicking method already returns an array of PickInfo objects.